libautoscoper/src/Camera.cpp

```diff
@@ -442,8 +442,10 @@ namespace xromm
     if (fx == 0 || fy == 0) {
       throw std::runtime_error("Invalid camera parameters (fx or fy is zero)");
     }
+    // viewport_[0] and viewport[0]_ approximately describe the x and y viewport coordinates of the (0,0) pixel
```

Comment:
```suggestion
    // viewport_[0] and viewport[0]_ describe the x and y viewport coordinates of the (0,0) pixel
```

Reply:
What is the rational for using "approximately" ?

Reply:
Since the principal point (`cx`,`cy`) is not always halfway between 0 and size for each dimension. The computed point may not represent exactly the pixel (0,0).


![PXL_20231109_201327915](https://github.com/BrownBiomechanics/Autoscoper/assets/30351234/83ae46c3-c55b-4738-9685-7d34b2539dcd)